src/image-metal.ks

```diff
@@ -0,0 +1,4 @@
+# If you add something here, also update image-cloud.ks.
+# prjquota is for quota enablement for containers: https://bugzilla.redhat.com/show_bug.cgi?id=1658386
+# rw and $ignition_firstboot are used by https://github.com/coreos/ignition-dracut/
+bootloader --append="rootflags=defaults,prjquota rw $ignition_firstboot"
```

Comment:
are we  going to attempt to have the same kernel command line across all images in the future or is that something where we'll have the defaults and then platform specific extras? mostly asking about this because of the intersection with https://github.com/ostreedev/ostree/issues/479

Reply:
I think having the same kargs across metal and cloud would require e.g. suppressing `net.ifnames` in Ignition or something like that right?

If this lands I can do a further cleanup that avoids the duplication of the kargs.

Reply:
+1